includes/backup.js

```diff
@@ -63,27 +63,29 @@ module.exports = function(dbClient, options, targetStream, ee) {
         return [
           allDocsGenerator(dbClient, options)
         ];
-      } else if (options.resume) {
-      // Resuming a backup, get the log file summary
-      // (changes complete and remaining batch numbers to backup)
+      } else {
+        // Full backup, we'll return a sream over a completed changes log file
```

Comment:
```suggestion
        // Full backup, we'll return a stream over a completed changes log file
```

Reply:
oh not again, I've really got to put some new shims in my keyboard 😄